JavaScript/Programmes/Symbol.js

```diff
@@ -1,16 +1,17 @@
-const _password = Symbol("password");
 class User {
-  constructor(unsername, password) {
-    this.username = this.username;
-    this[_password] = password;
-  }
+  constructor(username, password) {
+  this.username = username;
+  let _password = Symbol(password)
+
 
-  getPassword() {
-    return this[_password];
+  this.getPassword = () => {
```

Comment:
this reference is not required here.

Reply:
We won't be able to access the _password from outside the constructor, that's why the function is given as this.getPassword.

Reply:
That's a syntax error. this method is not inside a constructor (not a static method) and you can't create an instance method using this.methodName in a class.